tools/checkstyle/checkstyle.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -euo pipefail
 
+CI_INPUT="${1:-}"
```

Comment:
nit: maybe check `[[ "$CI" == true ]]` instead? we set this env var for workflows. Seems like it could simplify flag parsing etc

Reply:
Done - good idea